### DISCUSSION TITLE
Flood risk zones

### DISCUSSION THREAD

**Discussion by colmjude** [digital-land/data-standards-backlog#46] [Planning consideration]
# Flood risk zones

Zones in the UK are given a flood risk level. Usually 1, [2](#5) or [3](#6), with 3 being the most at risk.

We recently heard about flood risk zone 3bs. We think these zones are designated by Local planning authorities but we need to find out for sure.

Instead of treating each zone level as a separate thing we decided to combine them into one 'planning consideration' called Flood risk zones.

## Why (are we looking at this)

Flood risk areas are one of the things planning officers look at when assessing a planning application. They are a constraint that are tackled in successful applications.

For anyone to innovate and create software and services that help users with planning applications they will need to be able to access up-to-date and accurate flood risk zone data.

## What we've looking for

We want to know more about:

* how flood risk zones are used?
* what decisions are made using flood risk zones?
* if you use the flood risk zone data now, how do you use it?
* is there anything missing?
* what do you know about flood risk zone 3bs - how are they create, by whom and how are they used?
* is there anything else you think we should know about flood risk zones?

**Comment by kieranwint** [comment]
Through some of the initial research on Flood Risk Zone 1, we understand that any areas not covered by Flood Risk Zones 2 and 3 (3a/3b) are classed as Zone 1 as this is the lowest risk category. Would anyone be able to clarify this?

As well as the above, our research has led us to believe that local planning authorities hold the data regarding which zone 3 areas are designated as zone 3a and zone 3b. Would any LPAs be able to confirm this and also whether there is different criteria across different LPAs for the zone 3 split?

**Comment by skipster2k2** [comment]
Hi @colmjude I've just started a project with DEFRA looking at benchmarking hydrological methods, it would be great to have a chat about what you've learnt about how flood risk assessments are created if possible. My email is ian.ames@defra.gov.uk

**Reply by mattlucht** [reply]
Hi Ian – I just spotted this, and apologies if we've not gotten back to you yet. It'd be good to talk, I'll drop you and @colmjude a message and we can set something up!

**Comment by greg-slater** [comment]
The Environment Agency have published a [updated dataset](https://environment.data.gov.uk/dataset/04532375-a198-476e-985e-0579a0a11b47), which incorporates some methodology changes. There is still no persistent identifier in the updated dataset, and the field we used for a reference in the last dataset (objectid) doesn’t link entities across the datasets. Some questions that need answering before we can add the data:

* should we be adding new datasets without a persistent identifier?
* if so, is end-dating the existing entities and generating new ones for the updated dataset a reasonable approach?
* are there any follow up considerations if we end-date the entirety of an existing dataset? e.g. do the map and API queries default to returning only entities without an end-date, and if not should they?

**Reply by psd** [reply]
It's not ideal, but we can collect data and match it to entities without persistent identifiers, it's just more work for us to match and assign the entities using lookups scoped to a resource.

**Reply by psd** [reply]
If the entities are different things, then we could add an end-date. We shoudn't remove data like this. Agreed the map view needs work. I think most users expect to just see current data. We could add some UI to also show historical data.

**Reply by greg-slater** [reply]
I've done a [quick bit of analysis](https://github.com/digital-land/jupyter-analysis/tree/main/analysis/2025-04_pdp_flood_risk) between the two datasets, and the new 2025 data does appear to be quite different, though there are some records which have identical geometries. The most significant difference is that there are many more records in the new dataset; ~12 million, compared to ~78 thousand in the 2023 data. 

I tested matching flood risk zone 2 geometries in a few sample areas and found that it was mostly just a handful of the very small zones that have the same geometries across datasets. For an area around Bath 4/41 2023 records matched to 4/1,228 2025 records.

So we've been trying to weigh up the pros and cons of each approach, considering the relatively small number of records which would be linked to the same entity number. One of the considerations if we match what we can is that we don't currently have an agreed long term method to add end-dates for individual entities, just the [test](https://github.com/orgs/digital-land/discussions/355) we tried with brownfield land. And we're aiming to start exploring some different options for this soon which could make it easier to create and maintain end-dates. End-dating the entirety of the existing dataset would avoid this challenge as we could use the default-value config for the existing resource.

**Comment by drewhardy77** [comment]
Flood Zone 3b is a **local** designation, also referred to as the 'functional floodplain'. See [Flood Risk and coastal change guidance](https://www.gov.uk/guidance/flood-risk-and-coastal-change) for further details. These are produced by local planning authorities as part of their Strategic Flood Risk Assessments (SFRAs) and are not derivable from national probabilistic mapping, though will often share geometries. Obtaining this data would require open access to SFRAs.

**Reply by drewhardy77** [reply]
Further confirmation on [this Defra Data Services](https://environment.data.gov.uk/support/faqs/778338325) page

```
LPAs are responsible for producing maps showing the extent of functional floodplain (Flood Zone 3b) as part of Strategic Flood Risk Assessment(s). The Flood Map for Planning does not identify functional floodplain.
```

Flood Zone 3b has now been [made into a discrete discussion](https://github.com/digital-land/data-standards-backlog/discussions/322) - the discussion here can concentrate on Environment Agency designated Flood Zones
